### ISSUE TITLE
[FEEDSTOCK REQUEST] Huggingface Accelerate 

### ISSUE THREAD

**Issue by asmith26** [open-ce/open-ce#746]
**Describe the package you'd like added**
The Huggingface Accelerate lib provides _"A simple way to train and use PyTorch models with multi-GPU, TPU, mixed-precision."_

**Describe how this package fits in with the project**
A number of other huggingface packages have already been added to this project.

**Additional context**
This package does seem to make it very easy to used multiple devices: https://github.com/huggingface/accelerate#easy-to-integrate

**Comment by npanpaliya** [comment]
Thanks @asmith26 ! We will see if it can be included in our coming release.

**Comment by cdeepali** [comment]
@asmith26, I see that Anaconda has recently released `huggingface_accelerate v0.15.0` for both PPC and x86. 

```
$ conda search huggingface_accelerate
Loading channels: done
# Name                       Version           Build  Channel
huggingface_accelerate          0.15.0 py310h06a4308_0  pkgs/main
huggingface_accelerate          0.15.0  py37h06a4308_0  pkgs/main
huggingface_accelerate          0.15.0  py38h06a4308_0  pkgs/main
huggingface_accelerate          0.15.0  py39h06a4308_0  pkgs/main
```

**Comment by asmith26** [comment]
Many thanks @cdeepali for the info, can confirm `mamba install huggingface_accelerate` installs the latest version for me.
